scripts/cloudant/manage_cloudant.py

```diff
@@ -8,6 +8,12 @@
 Auth = namedtuple('Auth', 'username password')
 
 
+class AskRun(namedtuple('AskRun', 'ask run')):
+    def ask_and_run(self):
+        ask_user(self.ask())
```

Comment:
do `ask` and `run` end up being bound methods? if so then `self` will be passed to them and it looks like they don't accept any params in the places they're defined